syntax/internal/stdlib/stdlib.go

```diff
@@ -376,6 +377,23 @@ var combineMaps = value.RawFunction(func(funcValue value.Value, args ...value.Va
 		}
 	}
 
+	// Validate args[3]
+	passthrough_lhs := false
```

Comment:
```suggestion
	passthroughLHS := false
```
Underscores are usually not used in variable names in go